cypress-tests/cypress/support/commands.js

```diff
@@ -2365,26 +2310,38 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add("stepUp", (stepUpBody, globalState, value) => {
-  const key = `step_up_enabled_${globalState.get("merchantId")}`;
-  stepUpBody.key = key;
-  stepUpBody.value = value;
+Cypress.Commands.add("updateConfig", (configType, configData, globalState, value) => {
+  let url;
+  let body;
+  
+  switch (configType) {
+    case 'autoRetry':
+      url = `${globalState.get("baseUrl")}/configs/autoRetry`;
+      body = { ...configData, enable: value }; // enable or disable autoRetry
+      break;
+    case 'maxRetries':
+      url = `${globalState.get("baseUrl")}/configs/maxRetries`;
+      body = { ...configData, retries: value }; // set max retries
+      break;
+    case 'stepUp':
+      url = `${globalState.get("baseUrl")}/configs/stepUp`;
+      body = { ...configData, connectors: value }; // step-up configurations
+      break;
+    default:
+      throw new Error('Invalid config type');
+  }
 
   cy.request({
-    method: "POST",
-    url: `${globalState.get("baseUrl")}/configs/${key}`,
+    method: 'POST',
+    url: url,
     headers: {
       "Content-Type": "application/json",
       "api-key": globalState.get("adminApiKey"),
     },
-    body: stepUpBody,
+    body: body,
     failOnStatusCode: false,
   }).then((response) => {
-    logRequestId(response.headers["x-request-id"]);
-
-    if (response.status === 200) {
-      expect(response.body).to.have.property("key").to.equal(key);
-      expect(response.body).to.have.property("value").to.equal(value);
-    }
```

Comment:
In the `response`, we get what we send in the `request` when the response status is `200`. So it is better have the same..

Reply:
add this back:

<img width="787" alt="image" src="https://github.com/user-attachments/assets/81824534-a5d9-40f7-bc84-968c060c7850">